includes/html/common/availability-map.inc.php

```diff
@@ -195,6 +195,7 @@
         $sql .= " ORDER BY `".$deviceOrderBy."`";
 
         $temp_output = array();
+        $updowntime = "";
 
         foreach (dbFetchRows($sql, $param) as $device) {
```

Comment:
wouldn't it make more sense to do `$updowntime = "";` here to reset it's content "every round"?

Reply:
fixed. Thanks